src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/comment/services/LemmyCommentReplyService.java

```diff
@@ -46,40 +56,50 @@ private CommentReplyView.CommentReplyViewBuilder commentReplyViewBuilder(
     final Comment lemmyComment = conversionService.convert(commentReply.getComment(),
         Comment.class);
 
-    final CommentAggregates commentAggregates = conversionService.convert(
-        commentReply.getComment().getCommentAggregate(), CommentAggregates.class);
+    final CommentAggregates commentAggregates = conversionService.convert(commentReply.getComment()
+        .getCommentAggregate(), CommentAggregates.class);
 
-    final Post lemmyPost = conversionService.convert(commentReply.getComment().getPost(),
-        Post.class);
+    final Post lemmyPost = conversionService.convert(commentReply.getComment()
+        .getPost(), Post.class);
 
-    final Community lemmyCommunity = conversionService.convert(
-        commentReply.getComment().getCommunity(), Community.class);
+    final Community lemmyCommunity = conversionService.convert(commentReply.getComment()
+        .getCommunity(), Community.class);
 
-    final int personVote = commentLikeService.getPersonCommentVote(Person,
+    final int personVote = commentLikeService.getPersonCommentVote(person,
         commentReply.getComment());
 
     final CommentReply lemmyCommentReply = conversionService.convert(commentReply,
         CommentReply.class);
 
     final boolean creatorBannedFromCommunity = linkPersonCommunityService.hasLink(creator,
```

Comment:
There is a lot of other DB activity above that isn't necessary if the user is banned (eg looking up their vote on the comment). Maybe these checks should happen earlier in the method?